src/main/java/com/example/parser/service/PermitService.java

```diff
@@ -0,0 +1,44 @@
+package com.example.parser.service;
+
+import com.example.parser.dto.ResponseFromParsing;
+import com.example.parser.entity.PermitForEmissionsOfPollutants;
+import com.example.parser.repository.PermitRepository;
+import com.example.parser.utils.Parser;
+import org.json.simple.parser.ParseException;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class PermitService {
```

Comment:
Service must contains endpoint processing logic. In another case it's not needed at all. Examle:
1) call DownloadFile code (singleton service or utility class) to download a file and save somewhere in temp files;
2) call Parser code (from appropriate singleton service) to parse downloaded data;
3) post-process parsed data (in case it's needed);
4) call repo code to save in database;
5) prepare endpoint response;

Reply:
done